modules/extensions/src/path-style/path-style.js

```diff
@@ -25,7 +25,8 @@ import {dist} from 'gl-matrix/vec3';
 const defaultProps = {
   getDashArray: {type: 'accessor', value: [0, 0]},
   getOffset: {type: 'accessor', value: 0},
-  dashJustified: false
+  dashJustified: false,
+  allowPickGap: false
```

Comment:
Preferably the prop starts with `dash` so that it's clear that it's from this extension. e.g. `dashGapPickable`?